docker/build_scripts/build.sh

```diff
@@ -48,11 +48,11 @@ mkdir -p /opt/python
 build_cpythons $CPYTHON_VERSIONS
 rm -rf /usr/local/ssl
 
-# Install patchelf and auditwheel (latest)
-curl -sLO http://nixos.org/releases/patchelf/patchelf-0.9/patchelf-0.9.tar.gz
-tar -xzf patchelf-0.9.tar.gz
-(cd patchelf-0.9 && ./configure && make && make install)
-rm -rf patchelf-0.9.tar.gz patchelf-0.9
+# Install patchelf and auditwheel (latest with unreleased bug fixes)
+curl -sLO http://nipy.bic.berkeley.edu/manylinux/patchelf-0.9njs2.tar.gz
```

Comment:
checking the hash like we do with openssl would be even more secure than https -- in addition to protecting against attacks on the data in transit, it also protects against someone compromising `nipy.bic.berkeley.edu`.

it's always hard to judge how seriously to take these kinds of security risks -- I guess anyone who compromises `nipy.bic.berkeley.edu` can probably add trojans to all kinds of fun stuff that @matthew-brett builds and then distributes all over the place :-). But I'm a little nervous about this docker image in particular, because srsly anyone who can sneak code into here can pull of a pretty apocalyptic security breach -- there are a lot of linux systems in this world that use `pip install`, and probably all of them are going to effectively end up running arbitrary code picked by anyone who compromises this docker image. So taking some simple precautions to reduce our exposure seems like reasonable prudence :-)